episodes/managing_issues.md

```diff
@@ -84,58 +67,108 @@ I recommend the following labels:
+Without a project management tool/framework, it can be hard to keep track of what’s done, or what needs doing, and particularly difficult to convey that to others in the team or share the responsibilities.
+
+Different tools and platforms exist to help you with project planning and management -- 
+for example, [Asana](https://www.asana.com), [Trello](https://www.trello.com), [Miro](https://miro.com/), GitHub itself, even sticky notes on a white board. 
+Different tools will work best for different teams of collaborators, but making sure you use any tool to plan and manage work on your lesson development project is better than making it up as you go along.
+
+### Milestones
+Carpentries lessons are developed in GitHub so it is convenient (but not a must) to use its features to keep track of different components of the project and their current status. 
+
+GitHub offers two features for task/project management: _Projects_ and _Milestones_.
+Projects offer a more advanced set of functions and are particularly useful when you need to work on related tasks spread across multiple repositories.
+[Milestones](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones) are comparatively simple and a great place to start, especially if your efforts are focused on a single repository.
+The focus will be on Milestones in this session but we recommend that you [learn more about Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) if you want to take advantage of the full range of GitHub's features for project management.
+
+Milestones are a way of organising issues on your project into smaller units of work (e.g. deliverables) that have deadlines and progress of which needs to be closely tracked (e.g. release management). 
+They are good for managing time-bound, structured iterations and working in short sprints on batches of issues, which makes them a great fit for [the iterative process of lesson development recommended in Collaborative Lesson Development Training](https://carpentries.github.io/lesson-development-training/lesson-design.html). 
+
+Here are some helpful features of Milestones:
+
+* Add a title and description to the Milestone, e.g. _"Issues to be addressed before October pilot workshop"_ or _"Accessibility Enhancement Drive"_.
+* The Milestone represents a subset of issues on your repository, e.g. the things you want to work on next or a set of issues that share a theme.
+  Issues are assigned to a Milestone in a similar way to adding a label.
+* The Milestone can be given a target completion date.
+* A progress bar indicates how close you are to completing the Milestone.
+* The order of issues listed in the Milestone can be adjusted e.g. to indicate task priority to yourself and your collaborators.
```

Comment:
```suggestion
* You can add a title and description to the Milestone, e.g. _"Issues to be addressed before October pilot workshop"_ or _"Accessibility Enhancement Drive"_.
* The Milestone represents a subset of issues on your repository, e.g. the things you want to work on next or a set of issues that share a theme.
  Issues are assigned to a Milestone in a similar way to adding a label.
* The Milestone can be given a target completion date.
* A progress bar indicates how close you are to completing the Milestone in terms of the number of completed issues verus the total number of issues.
* The order of issues listed in the Milestone can be adjusted manually by dragging the issue up and down the list, e.g. to indicate task priority to yourself and your collaborators.
```

Reply:
Not sure if this extra text is helpful - perhaps for people who have not seen this before the extra explanations might be helpful. Will leave to @tobyhodges to decide.